Demo/Application/Base/Settings/BraintreeDemoSettings.swift

```diff
@@ -35,28 +35,19 @@ class BraintreeDemoSettings: NSObject {
 
     static var currentEnvironmentName: String {
         switch currentEnvironment {
-        case .sandbox:
+        case .sandbox, .custom:
```

Comment:
Right now I have this set up to hit sandbox since that seemed like what we would hit the most. Could also consider building logic to parse the first part of the environment from the tokenization key if folks think we may also want to use this for prod testing. Thoughts?

Reply:
Worth noting on the Braintree Gateway side stage routes through sandbox as well.